dashboards-observability/.cypress/utils/event_constants.js

```diff
@@ -138,3 +138,17 @@ export const renderLineChart = () => {
   querySearch(TEST_QUERIES[5].query, TEST_QUERIES[5].dateRangeDOM);
   cy.get('[data-test-subj="configPane__vizTypeSelector"] [data-test-subj="comboBoxInput"]').type('Line').type('{enter}');
 };
+
+export const renderHistogramChart = () => {
```

Comment:
this common method should be part of `1_event_analytics.spec.js` file instead of constants

Reply:
If we move all the methods of other charts too in here, the file length would be too long. Do we still want to do it?